gh-135001: Explicitly specify the encoding parameter value of `calendar.HTMLCalendar` as 'utf-8' (#135002)
python/cpython: Doc/library/calendar.rst

      3) specifies the number of months per row. *css* is the name for the
      cascading style sheet to be used. :const:`None` can be passed if no style
      sheet should be used. *encoding* specifies the encoding to be used for the

>       output (defaulting to ``utf-8``).

Reviewer: ```suggestion
      output (defaulting to ``'utf-8'``).
```

      .. versionchanged:: next

>          If *encoding* is None, utf-8 is used by default.

Reviewer: ```suggestion
         If *encoding* is None, ``'utf-8'`` is used by default.
```

   The encoding to use for output.
   :option:`--encoding` is required if :option:`--locale` is set.

>    Defaults to None.

Reviewer: ```suggestion
```

.. versionchanged:: next

>    If *encoding* is None, utf-8 is used by default.

Reviewer: ```suggestion
   If no :option:`--encoding` is supplied, ``'utf-8'`` is used by default.
```
